src/Extensions/Logging/Internal/Replayable/ReplayibleActivity.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace Microsoft.Omex.Extensions.Logging.Replayable
+{
+	internal class ReplayableActivity : Activity
```

Comment:
Filename and classname aren't aligned. The classname is the correct spelling

Reply:
Fixed, thank you